addons/hr_expense/static/src/scss/hr_expense.scss

```diff
@@ -19,6 +19,10 @@
     }
 }
 
+.o_kanban_view .o_cp_bottom_left:has(.o_button_create_report) {
```

Comment:
Added the pseudo-class to avoid impacting other kanban control panels (as was happening in test_knowledge_kanban_command_tour)